version.go

```diff
@@ -2,5 +2,5 @@ package redis
 
 // Version is the current release version.
 func Version() string {
-	return "9.6.1"
+	return "9.7.0-beta.1"
```

Comment:
Typo? 
Also, we have already released a beta version in this format: 9.7.0-b1, so maybe we want to stick to this format

Reply:
I guess we have to stick to the conventions in go community

https://go.dev/doc/modules/version-numbers

Reply:
Previously the format was broken, f.e 9.6.0b1